lib/parsers/gjs-gts-parser.js

```diff
@@ -0,0 +1,498 @@
+const ContentTag = require('content-tag');
+const glimmer = require('@glimmer/syntax');
+const DocumentLines = require('../utils/document');
+const { visitorKeys: glimmerVisitorKeys } = require('@glimmer/syntax');
```

Comment:
yes, no hacky require anymore 🚀

Reply:
What version is in use here?
Current release of glimmer/syntax is ESM only 😅